.github/workflows/ci.yml

```diff
@@ -143,27 +131,22 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install Rust stable
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
+        id: toolchain
```

Comment:
why is this needed?

Reply:
Not a particular reason, only to avoid double stable toolchain specification, this info is already inside `id` task object

If you prefer I can remove it and add `cargo +stable` below